slashcommand.go

```diff
@@ -31,3 +31,70 @@ func (c *SlashCommandService) Unregister(teamID, trigger string) error {
 func (c *SlashCommandService) ExecuteSlashCommand(command *model.CommandArgs) (*model.CommandResponse, error) {
 	return c.api.ExecuteSlashCommand(command)
 }
+
+// CreateCommand creates a server-owned slash command that is not handled by the plugin
+// itself, and which will persist past the life of the plugin. The command will have its
+// CreatorId set to "" and its PluginId set to the id of the plugin that created it.
+//
+// Minimum server version: 5.28
+func (c *SlashCommandService) CreateCommand(command *model.Command) (*model.Command, error) {
```

Comment:
Unrelated to this PR, but I'd love to do a pass on nomenclature /in this repository/ since all the `Command` parts seem to stutter given the scoping to `SlashCommandService`. i.e. it could just be `api.SlashCommand.Create` vs. `api.SlashCommand.CreateCommand`.

Reply:
/shrug May as well do it now. :)